spec/user_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
Please add a test for `#recent_three_posts` on the User model as it's a requirement to test custom methods in the model.